doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -983,6 +990,27 @@ To change service parameters, follow these steps.
 3.  In the window that opens, configure the service parameters. To apply the selected parameter to all services described in the same service description, select the checkbox adjacent to this parameter in the **Apply to All in WSDL** column. To apply the configured parameters, click **OK**.
 
 
+### 6.7 Managing REST Endpoints
+
+**Access rights:** [Service Administrator](#xroad-service-administrator)
+
+REST type service descriptions can contain API endpoints. The purpose of the endpoints is more fine-grained access control. More about that in chapter [7 Access Rights](#7-access-rights).
+
+When URL type of the REST service is an OpenAPI 3 description, endpoints are parsed from the service description automatically. Additionally manual endpoints can be added as needed. When URL type is REST API base path, all the endpoints need to be created manually.
```

Comment:
When URL type of the REST service is an OpenAPI 3 description, endpoints are parsed from the service description automatically. These endpoints cannot be manually updated or deleted. Additionally manual endpoints can be added as needed. When URL type is REST API base path, all the endpoints need to be created manually. Manually created endpoints can also be edited and deleted as needed.

Reply:
Ok